### PR TITLE
Add initial project

### PR DESCRIPTION
**Pull request by kevinetienne**

### CODE REVIEW

**Comment by kevinetienne** [comment]
@Ian-Foote can you review please?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+language: python
+python:
+  - 3.4
+  - 3.5
+notifications:
+  email: false
+env:
+  matrix:
+    - DJANGO='django>=1.8,<1.9'
```
Comment (LilyFirefly) [inline .travis.yml]: `~=1.8.12`

What about `1.9`?

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[flake8]
+max-line-length = 90
+max-complexity = 10
+statistics = true
+application-import-names = user_deletion
+import-order-style = smarkets
```
Comment (LilyFirefly) [inline setup.cfg:6]: Don't we usually use `google`? Is this better?

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+[coverage:run]
+source = user_deletion
+omit = *tests*
```
Comment (LilyFirefly) [inline setup.cfg:10]: Maybe add `[coverage:report]` too?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+	@echo "Usage:"
+	@echo " make release | Release to pypi."
+	@echo " make test | Run the tests."
+
+release:
+	python setup.py register sdist bdist_wheel upload
+
+test:
+	@coverage run ./tests/run.py
+	@coverage report --show-missing
```
Comment (LilyFirefly) [inline Makefile]: `--show-missing` can be moved into `setup.cfg`

File: tests/run.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+from django.conf import settings
+from django.test.runner import DiscoverRunner
+
+
+BASEDIR = os.path.dirname(os.path.dirname(__file__))
+
+settings.configure(
+    DATABASES={
+        'default': dj_database_url.config(
+            default='sqlite://{}/user_deletion.db'.format(BASEDIR),
```
Comment (LilyFirefly) [inline tests/run.py:16]: Is there a reason not to use postgres?

File: .travis.yml (hunk 1 above)
Comment (kevinetienne) [inline .travis.yml]: oh yeah 1.9 is a good idea

File: setup.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    packages=find_packages(),
+    include_package_data=True,
+    version=version,
+    license='BSD',
+    description='Management commands to notify and delete inactive django users',
+    classifiers=(
+        'Development Status :: 1 - Planning',
+        'Framework :: Django',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
```
Comment (LilyFirefly) [inline setup.py:18]: ~~Do we have a `LICENSE` file?~~

File: setup.cfg (hunk 2 above)
Comment (kevinetienne) [inline setup.cfg:6]: `smarkets` has been added in https://github.com/public/flake8-import-order/pull/57

> smarkets - style as google only with import statements before from X import

File: tests/run.py (hunk 5 above)
Comment (kevinetienne) [inline tests/run.py:16]: No reason, as I just need to notify and delete users, I thought it was enough, what do you think?

Comment (LilyFirefly) [inline tests/run.py:16]: I guess it doesn't matter much. +0 for postgres for consistency.

File: setup.cfg [hunk 7]
```diff
@@ -8,3 +8,7 @@ import-order-style = smarkets
 [coverage:run]
 source = user_deletion
 omit = *tests*
+
+[coverage:report]
+show_missing = True
+fail_under = 100
```
Comment (KevinEtienne) [inline setup.cfg]: seems like `fail_under` fails on my local machine, it sounds like a bug, I'm going to remove it for now.

File: setup.cfg [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+application-import-names = user_deletion
+import-order-style = smarkets
+
+[coverage:run]
+source = user_deletion
+omit = *tests*
+
+[coverage:report]
+show_missing = True
+fail_under = 100
```
Comment (kevinetienne) [inline setup.cfg]: `fail_under=100` fails on my local machine. It sounds like a bug, I'm going to remove it.

File: setup.cfg (hunk 3 above)
Comment (kevinetienne) [inline setup.cfg:10]: Added in https://github.com/incuna/django-user-deletion/pull/1/commits/f88d25d881db037a6c9f5e64e7861b55c00fe7b1

**Pull request merged**